Editor/EditorCore/SmoothGroupEditor.cs

```diff
@@ -507,7 +507,7 @@ void OnGUI()
                 global::UnityEditor.ProBuilder.UI.EditorGUILayout.EndRow();
 
                 if (GUILayout.Button("Open Documentation"))
-                    Application.OpenURL("http://procore3d.github.io/probuilder2/toolbar/tool-panels/#smoothing-groups");
+                    Application.OpenURL("https://docs.unity3d.com/Packages/com.unity.probuilder@4.2/manual/workflow-edit-smoothing.html");
```

Comment:
Is there a `@latest` or similar that we can use instead of hard-coding 4.2?

Reply:
Yes, it is done in 7d05b91